index.html

```diff
@@ -2360,69 +2367,124 @@ <h2 id="OfflineAudioCompletionEvent">
           value explicitly. <em>block-size</em> is defined to be 128
           sample-frames which corresponds to roughly 3ms at a sample-rate of
           44.1 kHz.
-        </p><a>AudioNode</a>s can be created in two ways:
+        </p>
+        <p>
+          <a>AudioNode</a>s can be created in two ways: by using the
+          constructor for this particular interface, or by using the
+          <dfn>factory method</dfn> on the <a>BaseAudioContext</a> or
+          <a>AudioContext</a>.
+        </p>
+        <p>
+          The <a>BaseAudioContext</a> passed as first argument of the
+          constructor of an <a>AudioNode</a>s is called the <dfn id=
+          "associated">associated <a>BaseAudioContext</a></dfn> of the
+          <a>AudioNode</a> to be created. Similarly, when using the factory
+          method, the <a>associated <code>BaseAudioContext</code></a> of the
+          <a>AudioNode</a> is the first argument of the factory method.
+        </p>
+        <p>
+          To create a new <a>AudioNode</a> of a particular type <var>i</var>
+          using its constructor, with a <a>BaseAudioContext</a> <var>c</var> as
+          first argument, and an <a>associated option object</a>
+          <var>option</var> as second argument, from the <a href=
+          "https://html.spec.whatwg.org/multipage/webappapis.html#concept-relevant-global">
+          relevant global</a> of <var>c</var>, execute these steps:
+        </p>
+        <ol>
+          <li>Let <var>o</var> be a new object of type <var>i</var>.
+          </li>
+          <li>
+            <a href="#audionode-constructor-common">Initialize the common
+            part</a> of <var>o</var>, with <var>c</var> and <var>option</var>
+            as arguments.
+          </li>
+          <li>Return <var>o</var>
+          </li>
+        </ol>
+        <p>
+          To create a new <a>AudioNode</a> of a particular type <var>i</var>
+          using its <a>factory method</a>, called on a
+          <var>BaseAudioContext</var> <var>c</var>, execute these steps:
+        </p>
+        <ol>
+          <li>Let <var>o</var> be a new object of type <var>i</var>.
+          </li>
+          <li>Let <var>option</var> be a dictionnary of the type associated to
+          the interface associated to this factory method.
+          </li>
+          <li>For each parameter passed to the factory method, set the
+          dictionary member of the same name on <var>option</var> to the value
+          of this parameter.
+          </li>
+          <li>
+            <a href="#audionode-constructor-common">Initialize the common
+            part</a> of <var>o</var> with <var>c</var> and <var>option</var> as
+            arguments.
+          </li>
+          <li>Return <var>o</var>
+          </li>
+        </ol>
+        <p>
+          <dfn id="audionode-constructor-common">Initializing the common
+          part</dfn> of an object <var>o</var> of interface <var>i</var> that
```

Comment:
This talks about initializing the common parts like the number of inputs/outputs and channel parameters, and it does.  But then it also talks about the non-common parts like the node-specific options.

I think this needs to be split up into two pieces or just renamed to so that's not about initializing the common part.